src/stylesheets/footer.less

```diff
@@ -0,0 +1,18 @@
+@import 'colors.less';
+
+.footer {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 1rem;
+  color: @link-text;
+  background-color: @footer;
```

Comment:
instead of naming the color `footer`, would it be our secondary color (where the primary is the blue)?